feature_layers/feature-collection-layer-from-portal/README.metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "category": "Feature layers",
+    "description": "Combine feature tables with different geometries into a single layer.",
+    "ignore": false,
+    "images": [
+        "FeatureCollectionLayer.png"
```

This should also refer to the screenshot from this sample:
```suggestion
        "FeatureCollectionLayerFromPortal.png"
```